pkgs/development/tools/minizinc/ide.nix

```diff
@@ -1,19 +1,20 @@
-{ lib, stdenv, fetchFromGitHub, qtbase, qtwebengine, qtwebkit, qmake, makeWrapper, minizinc }:
+{ lib, mkDerivation, fetchFromGitHub, qtbase, qtwebengine, qtwebkit, qmake, minizinc }:
 let
-  version = "2.4.3";
+  version = "2.5.3";
 in
-stdenv.mkDerivation {
+mkDerivation {
```

Comment:
I am not sure if this breaks something.

Reply:
This change is what was required to make it work with Qt at all.

Reply:
Oh, yeah. It is required for QT.